src/scenes/Root/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  List,
+  ListItemIcon,
+  ListItemText,
+  ListSubheader,
+  makeStyles,
+  Menu,
+  MenuItem,
+  Paper,
+  SvgIconProps,
+  ThemeProvider,
+} from '@material-ui/core';
+import { Add, FolderOpen, Language, People } from '@material-ui/icons';
+import { ComponentType, FC, useState } from 'react';
+import * as React from 'react';
+import { ErrorButton } from '../../../components/ErrorButton';
+import { PlantIcon } from '../../../components/Icons';
+import { ListItemLink, ListItemLinkProps } from '../../../components/Routing';
+import { sidebarTheme } from './sidebar.theme';
+import { SidebarHeader } from './SidebarHeader';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  root: {
+    maxWidth: 248,
+  },
+  content: {
+    padding: spacing(0, 2),
+  },
+  createNewItem: {
+    margin: spacing(4, 2, 1),
+    width: `calc(100% - ${spacing(2 * 2)}px)`,
+  },
+}));
+
+export const Sidebar: FC = () => {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleClick = (e: any) => {
+    setAnchorEl(e.currentTarget);
+  };
+
+  const handleClose = () => setAnchorEl(null);
+
+  const createMenu = (
```

Comment:
Do you think we should we split these off into separate files?

Reply:
Yeah there's definitely a good chance we'll want to refactor this into a separate component. Let's wait and see an actual usage first.